__tests__/endpoints.test.js

```diff
@@ -131,8 +132,51 @@ describe("'/api/articles' endpoint", () => {
         })
     })
     describe("GET all comments for a specific article from '/api/articles/:article_id/comments' endpoint", () => {
-        test("", () => {
-
+        test("GET correct length array with comments listed in descending time order. Comments should be objects with certain properties", () => {
+            return request(app)
+            .get('/api/articles/9/comments')
+            .expect(200)
+            .then(({body}) => {
+                expect(Array.isArray(body.comments)).toBe(true)
+                expect(body.comments).toHaveLength(2)
+                body.comments.forEach(comment => {
+                    expect(comment).toMatchObject({
+                        comment_id: expect.any(Number),
+                        votes: expect.any(Number),
+                        article_id: expect.any(Number),
+                        body: expect.any(String),
+                        created_at: expect.any(String),
+                        author: expect.any(String),
+                    });
+                });
+                expect(body.comments[0].comment_id).toBe(1)
+                expect(body.comments[1].comment_id).toBe(17)
+            })
+        })
+        test("GET request to article with no comments returns 200 and empty array", () => {
+            return request(app)
+            .get('/api/articles/8/comments')
+            .expect(200)
+            .then(({body}) => {
+                expect(Array.isArray(body.comments)).toBe(true)
+                expect(body.comments).toHaveLength(0)
+            })
+        })
+        test("GET request to non-valid article_id returns 400 and message string detailing the error", () => {
+            return request(app)
+            .get('/api/articles/ruairi/comments')
+            .expect(400)
+            .then(({body}) => {
+                expect(body.msg).toBe('Article ID submitted is invalid datatype')
```

Comment:
You could choose to make this error massage a little less-informative, just something like "Bad request" would be sufficient, and prevents us from giving away too many details about the structure of your API